tests/test_filters.py

```diff
@@ -172,11 +172,11 @@ class SearchListView(generics.ListAPIView):
             search_fields = ('$title', '$text')
 
         view = SearchListView.as_view()
-        request = factory.get('/', {'search': '^\w{3}$'})
+        request = factory.get('/', {'search': '^\\w{3}$'})
```

Comment:
Would it make more sense to use the `r` modifier here for the string?

Reply:
yeah seems so